### ISSUE TITLE
Add more users & their password database , update function to check valid user

### ISSUE THREAD

**Issue by khushi0704** [khushi0704/UserLogin-system#4]

**Comment by insaneshri8** [comment]
I am New in Open source contribution can u help me to telling the exact problem . I can fix the issue but i need support to level up . Can u guide me?

**Comment by LinShien** [comment]
@khushi0704  Maybe we should use a remote backend database like MySQL or sqlite in Azure like platform to store all the usernames and passwords. Because stores all the login information in client side is not practical in real application.

**Comment by khushi0704** [comment]
> @khushi0704 Maybe we should use a remote backend database like MySQL or sqlite in Azure like platform to store all the usernames and passwords. Because stores all the login information in client side is not practical in real application.

@LinShien Yes , You are right , I am going to implement that . Initially I was just experimenting with the code but I intended to make use of a Database Management system later. 
If you want you are free to play with the code however you like , I'm in for relavant changes . Be at ease. Happy Coding !

**Comment by khushi0704** [comment]
> I am New in Open source contribution can u help me to telling the exact problem . I can fix the issue but i need support to level up . Can u guide me?

welcome to the Open Source community ! @insaneshri8 . The exact problem is , as you can see , this is a user-login system build in c++ using OOP & file handling concepts , so now I want to store usernames and their respective password in a more accurate and relavant way , for suppose , storing them in text files is a bad example (any credentials shouldn't be stored publicly) , so I am going to create a database like mysql where all credentials would be stored and link it to the .cpp file.
You're welcome to play , experiment with the code however you like , just to remind , it should be relevant. Happy Coding !
